rules/utils/utils.js

```diff
@@ -97,6 +104,9 @@
 				return callObject.name === '$stateProvider';
 			}
 			return false;
-		}
+		},
+
+		scopeProperties: ['$id', '$parent', '$root', '$destroy', '$broadcast', '$emit', '$on', '$applyAsync', '$apply',
```

Comment:
This list is from the Angular docs. The style guide suggests moving event calls (like $emit and $broadcast) to services, but I'm allowing those right now. We can set this list to whatever you think is best.